play-json-tools/src/main/scala/com/evolutiongaming/util/JsonFormats.scala

```diff
@@ -285,4 +285,13 @@ object JsonFormats {
 
     def writes(x: NewNel[T]): JsValue = Json toJson x.toList
   }
+
+  implicit val UnitFormat = new Format[Unit] {
```

Comment:
Could you please add return type for public member ?

Reply:
Sure